scripts/build_script.sh

```diff
@@ -15,6 +15,7 @@ help()
     echo "h     Print this Help."
     echo "p     Generate a Debian package using CPack."
     echo "s     Suffix to append to version for Debian packaging."
+    echo "D     Pass CMake options to build command (e.i -DBUILD_TESTS=ON)"
```

Comment:
Do we pass `-D BUILD_TESTS=ON` or `-D -DBUILD_TESTS=ON`?

Reply:
How about passing `-t ON` rather than `-D BUILD_TESTS=ON`?

Reply:
This is for passing **custom** cmake arguments to the cmake call  so argument can be anything, we are just passing them to CMake. The syntax for passing CMake options are variables is -D<VARIABLE_NAME>=VALUE so we keep this sytnax here